docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -52,25 +52,80 @@ image::images/customize-event-renderer-page.png[Shows the Event Renderer page]
+*Summary tab*
+
+The *Summary* tab provides an overview of the alert and shows relevant threat intelligence details. You can use this information to gain an understanding of what caused the alert and to make decisions about handling it.
+
+The *Summary* tab always begins with a list of general alert details. These details are conveyed as a curated list of fields containing essential and available information about the alert. Within this section, you can find the alert's current status, locate the rule that produced the alert, learn the alert's severity, and more. You can use these details to start forming a sense of what caused the alert and then continue your investigation by checking out available threat intelligence on the alert.
+
+The {es-sec} app collects threat intelligence data for all alerts. Fields that show threat intelligence data are appended with an informational icon and placed after general alert details on the *Summary* tab.
+
+When gathering threat intelligence, the {es-sec} app  queries indicator data from the past 30 days to generate enrichments for the below fields.
+If these field values aren't found, the {es-sec} app does not create threat intelligence for the alert.
+
+NOTE: You can learn more about the query by going to the **Threat Intel** tab and clicking the **Inspect** icon next to threat you want to inspect.
+
+- `file.hash.md5`: The MD5 hash
+- `file.hash.sha1`: The SHA1 hash
+- `file.hash.sha256`: The SHA256 hash
+- `file.pe.imphash`: Imports in a PE file
+- `file.elf.telfhash`: Imports in an ELF file
+- `file.hash.ssdeep`: The SSDEEP hash
+- `source.ip`: The IP address of the source (IPv4 or IPv6)
+- `destination.ip`*: The event's destination IP address
+- `url.full`: The full URL of the event source
+- `registry.path`: The full registry path, including the hive, key, and value
+
+Additional threat intelligence is generated for indicator match alerts. These details are produced when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Any data that's duplicated is removed.
+
+*Threat Intel tab*
+
+Shows the number of threats linked the alert and displays them individually. Threat titles are formed using values from the `matched.field` and `matched.atomic` fields and threats appear in reverse chronological order, with the most recent at the top. Available `threat.indicator` and `source.event` data is displayed for each threat.
+
+If the alert does not contain threat intelligence data, the *Threat Intel* tab displays a message indicating that none is available and provides a link to Threat Intel module documentation.
+
+*Table tab*
+
+Shows the alert details in table format. Alert details are organized into field value pairs.
+
+*JSON View tab*
+
+Shows the alert details in JSON format.
+
+
+// * *Summary*: Shows an aggregated view of alert details, plus two types of threat intelligence data: indicator rule enrichments and investigation time enrichments.
```

Comment:
I think we can just delete these comments for now 👍; we can always resurrect them from git if we later need them.